tests/php/Shortcodes/ImageShortcodeProviderTest.php

```diff
@@ -93,4 +93,23 @@ public function testShortcodeHandlerAddsDefaultAttributes()
             ))
         );
     }
+
+    public function testShortcodeHandlerFailsGracefully()
+    {
+        $parser = new ShortcodeParser();
+        $parser->register('image', [ImageShortcodeProvider::class, 'handle_shortcode']);
+
+        $nonExistentImageID = 9999;
+        while(Image::get()->byID($nonExistentImageID)) {
```

Comment:
A little space after `while` is probably what's breaking the builds

Reply:
Ha. I ran `lint-clean`, but it's not defined for this package, so I figured I'd just let CI yell at me if it had a problem.